jquery.validate.unobtrusive.js

```diff
@@ -24,7 +24,7 @@
 
     function escapeAttributeValue(value) {
         // As mentioned on http://api.jquery.com/category/selectors/
-        return value.replace(/([!"#$%&'()*+,./:;<=>?@\[\\\]^`{|}~])/g, "\\\\$1");
+        return value.replace(/([!"#$%&'()*+,./:;<=>?@\[\\\]^`{|}~])/g, "\\$1");
```

Comment:
And we're sure that reverting this line doesn't break anything? What was it trying to fix, and why did we think this fix was correct at the time?